labelbox_dev/exceptions.py

```diff
@@ -0,0 +1,109 @@
+class LabelboxError(Exception):
```

Comment:
A lot of these exceptions are redundant and we should probably think through how best to handle them vs copying them over to the new code base. It makes it too easy to not fix if we re-use the same exceptions and I would argue that exception handling is pretty broken. 

Is there a plan for the exception hierarchy and types of exceptions that we will throw? This is a huge opportunity to fix this.

Reply:
I think the exception handling being broken is due to the fact that our backend doesn't report exceptions nicely. This is handled better in our REST controllers, as we are practicing throwing appropriate HTTP exceptions. With that said, instead of copying over all exceptions from SDK 3, maybe it'll be good to start with only the exceptions we need, and build from there, since we do have an opportunity to clean up possible redundant exceptions